framework/src/utils/BicubicInterpolation.C

```diff
@@ -53,26 +52,23 @@ BicubicInterpolation::sample(const ADReal & x1, const ADReal & x2) const
   return sampleInternal(x1, x2);
 }
 
-template <typename T>
-T
-BicubicInterpolation::sampleInternal(T x1, T x2) const
+template <class C>
+C
```

Comment:
Was this to get credit for more lines? :laughing: `T` is kind of the canonical template parameter. I don't see that `C` is more explanatory. Anyway I don't really care about this, but in general I prefer to use `T` instead of `C`

Reply:
C was chosen for a reason because T was failing actually. 
@spaubt2 do you remember why?

Reply:
I could see this as an issue if it was shadowing a class template parameter but I don't see how that would be the case here. Anyway this is not that important of a mystery to solve